Gemfile

```diff
@@ -15,6 +15,7 @@ gem 'bootstrap', '~> 4.0.0'
 gem 'jquery-rails'
 gem 'rspotify'
 gem 'devise'
+gem 'omniauth-spotify'
```

Comment:
Not sure why this commit is included in this PR. Not a blocker. Please be on top of your git flow.